Makefile

```diff
@@ -3,13 +3,13 @@ HOSTNAME=github.com
 NAMESPACE=infobloxopen
 NAME=b1ddi
 BINARY=terraform-provider-${NAME}
-VERSION=0.1.0
+VERSION=0.1.3
 OS_ARCH=linux_amd64
 
 default: install
 
 build:
-	go build -o ${BINARY}
+	go build -ldflags "-X main.terraformVersion=${VERSION}" -o ${BINARY}
```

Comment:
Do we need this val to be set for released terraform-porivder versions? 

In this case, `.goreleaser.yml`should be updated.

Reply:
Will undo this. As you have pointed me to the `.goreleaser.yml` file. Will make use of vars from `.goreleaser.yml` file while setting the header in the API calls